client/my-sites/earn/home.tsx

```diff
@@ -416,6 +425,13 @@ const Home = () => {
 		const defaultBody = translate(
 			'Share WordPress.com with friends, family, and website visitors. For every paying customer you send our way, you’ll both earn US$25 in free credits.'
```

Comment:
We should probably parse these currencies (`US$25`) with the formatter to ensure they are translated consistently, e.g., in el/de the symbol comes after, like `25 $`. Looking at some of the translations, they may not be quite right - sometimes spelled out like "25 dollars" or "25 USD" in non-English locales.

Same goes for the other strings in this file.